js/components/scrubberBar.jsx

```diff
@@ -507,6 +572,11 @@ class ScrubberBar extends React.Component {
         onMouseLeave={this.handleScrubberBarMouseLeave}
         onMouseMove={scrubberBarMouseMove}
       >
+        {controller.state.markers && controller.state.markers.list.length > 0 && (
+          <div id="oo-marker-container">
```

Comment:
Is it safe to use id here? What if there are 2 players on the page?

Reply:
nice catch!